src/activity_handlers.cpp

```diff
@@ -3956,6 +3956,8 @@ void activity_handlers::spellcasting_finish( player_activity *act, player *p )
         casting.make_sound( target );
     } else if( fx == "summon" ) {
         spell_effect::spawn_summoned_monster( casting, p->pos(), target );
+    } else if( fx == "translocate" ) {
+        spell_effect::translocate( casting, p->pos(), target, g->u.translocators );
```

Comment:
```suggestion
        spell_effect::translocate( casting, p->pos(), target, p->translocators );
```
Or do you expect `p` to be some NPC that happens to use the translocators of the player character?

Reply:
translocators are a member of `avatar`, as such `player` does not have this member. once i start looking at making it possible for npcs to cast spells, i will likely limit them much more than the player is.